backend/hitas/urls.py

```diff
@@ -55,6 +55,9 @@
 # /api/v1/housing-companies/{housing_company_id}/apartments/{apartment_id}/maximum-price
 apartment_router.register(r"maximum-prices", views.ApartmentMaximumPriceViewSet, basename="maximum-price")
 
+# /api/v1/housing-companies/{housing_company_id}/apartments/{apartment_id}/sale
+apartment_router.register(r"sales", views.ApartmentSaleViewSet, basename="apartment-sale")
```

Comment:
The URL will end up being `.../sales` (Please fix the comment for maximum price too 😅)

Reply:
Done.